src/encoded/regulome_indexer.py

```diff
@@ -340,16 +343,29 @@ def region(row):
            Extend this to get store and strand properties, although the bed files vary by type
         '''
         chrom, start, end = row[0], int(row[1]), int(row[2])
-        return (chrom, {
-                        'gte': start,
-                        'lt': end
-                    }
-                )  # Stored as BED 0-based half open
+        strand = '.'
+        if len(row) > 5 and row[5] in ['.', '+', '-']:
+            strand = row[5]
+        doc = {
+            'coordinates': {
+                'gte': start,
+                'lt': end
+            },
+            'strand': strand,
+        }  # Stored as BED 0-based half open
+        if len(row) > 3:
+            doc['value'] = row[3]
+        if len(row) > 4 and row[3] in ['.', '']:
+            doc['value'] = row[4]
+        return (chrom, doc)
 
     @staticmethod
     def snp(row):
         '''Read a SNP from an in memory row and returns chrom and document to index.'''
         chrom, start, end, rsid = row[0], int(row[1]), int(row[2]), row[3]
+        strand = '.'
+        if len(row) > 5 and row[5] in ['.', '+', '-']:
```

Comment:
I think we should write a warning if this condition is not met.

Reply:
Warning in log?

Reply:
good point.  This is in the indexer... I don't think it will write to STDERR/apache log.  Maybe @caseylitton knows.